.github/workflows/ci_cd.yml

```diff
@@ -668,98 +540,31 @@ jobs:
-      (needs.nuxt-build.result == 'success' || needs.nuxt-build.result == 'skipped') &&
-      github.event_name == 'push'
-    runs-on: ubuntu-latest
-    needs:
-      - test-ing
-      - test-api
-      - nuxt-build
-
-    steps:
-      - name: Checkout repository
-        uses: actions/checkout@v3
-
-      - name: Setup CI env
-        uses: ./.github/actions/setup-env
-        with:
-          github_token: ${{ secrets.GITHUB_TOKEN }}
-
-      - name: Compile documentation
-        uses: ./.github/actions/build-docs
-        # Docs will be located at `/tmp/docs`.
-
-      - name: Checkout repository at `gh-pages` branch
-        uses: actions/checkout@v3
-        with:
-          ref: gh-pages
-
-      - name: Preserve previews
-        run: mv _preview /tmp/docs/_preview
-
-      - name: Deploy
-        uses: peaceiris/actions-gh-pages@v3
-        with:
-          github_token: ${{ secrets.GITHUB_TOKEN }}
-          publish_dir: /tmp/docs
-          force_orphan: true
-          cname: docs.openverse.org
-
-      - name: Checkout repository # again, to enable cleaning
-        uses: actions/checkout@v3
-        if: always()
-
-  docs-preview:
-    name: Preview full-stack docs
+  emit-docs:
+    name: Emit full-stack docs
     # https://github.com/actions/runner/issues/491#issuecomment-850884422
     if: |
       !failure() && !cancelled() &&
+      (
+        github.event_name == 'push' ||
+        (
+          github.event_name == 'pull_request' &&
+          github.event.pull_request.head.repo.owner.login == 'WordPress' &&
+          github.actor != 'dependabot[bot]'
+        )
+      ) &&
       (needs.test-ing.result == 'success' || needs.test-ing.result == 'skipped') &&
       (needs.test-api.result == 'success' || needs.test-api.result == 'skipped') &&
-      (needs.nuxt-build.result == 'success' || needs.nuxt-build.result == 'skipped') &&
-      github.event_name == 'pull_request' && github.event.pull_request.head.repo.owner.login == 'WordPress' && github.actor != 'dependabot[bot]'
+      (needs.nuxt-build.result == 'success' || needs.nuxt-build.result == 'skipped')
     runs-on: ubuntu-latest
     needs:
       - test-ing
       - test-api
       - nuxt-build
     env:
-      REPO_OWNER: ${{ github.event.pull_request.head.repo.owner.login }}
       REPO_NAME: ${{ github.event.pull_request.head.repo.name }}
 
     steps:
-      - uses: peter-evans/find-comment@v2
-        id: preview-comment
-        with:
-          issue-number: ${{ github.event.pull_request.number }}
-          body-includes: Full-stack documentation
-
-      - uses: peter-evans/create-or-update-comment@v2
-        if: steps.preview-comment.outputs.comment-id != 0
-        with:
-          comment-id: ${{ steps.preview-comment.outputs.comment-id }}
-          edit-mode: replace
-          body: |
-            **Full-stack documentation**: _Updating_
-
-            (This comment will be automatically updated with the preview URL once it is ready)
```

Comment:
Does this mean we will no longer have the comment added before the docs are finished generating? I feel that it's helpful to have this comment beforehand, especially so that it's as close to the top of the PR as possible (and easy to find). I might be the only one though :sweat_smile:

Reply:
Having this here was not guaranteeing placement at the top of the PR because there are other workflow jobs that write comments (e.g. the bundle size check or the Playwright checks). that will take place either in parallel to CI or before the docs job. Not having this step only means the comment is written a couple of minutes later, not drastically shifting the position.